recipes/immer/all/conandata.yml

```diff
@@ -0,0 +1,3 @@
+sources:
+  "0.6.2":
+    url: "https://github.com/arximboldi/immer/archive/refs/tags/v0.6.2.tar.gz"
```

Comment:
```suggestion
    url: "https://github.com/arximboldi/immer/archive/refs/tags/v0.6.2.tar.gz"
    sha256: "c3bb8847034437dee64adacb04e1e0163ae640b596c582eb4c0aa1d7c6447cd7"
```

Reply:
@rilerez, did you forget to use this patch?